web_timeline/static/src/js/web_timeline.js

```diff
@@ -0,0 +1,514 @@
+/*---------------------------------------------------------
+ * Odoo web_timeline
+ * Copyright 2015 ACSONE SA/NV
+ * Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+ *---------------------------------------------------------*/
```

Comment:
I just hate those lines that give that big emphasis to the least important piece of the file. If you remove them, you'll make my eyes happier :smile:. Not required though.